insights/parsers/sysconfig.py

```diff
@@ -506,3 +509,52 @@ class VirtWhoSysconfig(SysconfigOptions):
         'A TEST'
     """
     pass
+
+
+@parser(Specs.ifcfg_static_route)
+class IfCFGStaticRoute(LegacyItemAccess, Parser):
+    """
+    IfCFGStaticRoute is a parser for the static route network interface
+    definition files in ``/etc/sysconfig/network-scripts``.  These are
+    pulled into the network scripts using ``source``, so they are mainly
+    ``bash`` environment declarations of the form **KEY=value**.  These
+    are stored in the ``data`` property as a dictionary.  Quotes surrounding
+    the value
+
+    Because this parser reads multiple files, the interfaces are stored as a
+    list within the parser and need to be iterated through in order to find
+    specific interfaces.
+
+    Sample configuration from a static connection in file ``/etc/sysconfig/network-scripts/rute-test-net``::
+
+        ADDRESS0=10.65.223.0
+        NETMASK0=255.255.254.0
+        GATEWAY0=10.65.223.1
+
+    Examples:
+
+        >>> conn_info.static_route
+        'test-net'
+        >>> conn_info.data['ADDRESS0']
+        '10.65.223.0'
+        >>> sorted(conn_info.data.items())
+        [('ADDRESS0', '10.65.223.0'), ('GATEWAY0', '10.65.223.1'), ('NETMASK0', '255.255.254.0')]
+
+    Properties:
+        static_route (str): static route file name derived from file path
```

Comment:
And then this `Properties` section can be removed.

Reply:
acknowledged, updated the code in the latest PR